README.md

```diff
@@ -16,3 +16,14 @@ You will also see any lint errors in the console.
 
 Builds a static copy of your site to the `build/` folder.
 Your app is ready to be deployed!
+
+## Deployments
+
+Latest production deployment 👉 https://app.radicle.xyz
+
+If you want to connect to pre-heartwood seed nodes, you can run the app locally
+via the legacy IPFS deployment. To do so you'll have to install and run either
+[Kubo](https://docs.ipfs.tech/how-to/command-line-quick-start) or the
+[IPFS Desktop](https://docs.ipfs.tech/install/ipfs-desktop) app.
```

Comment:
Can we just say "To do so, you'll have to run [IPFS](https://ipfs.tech)`. The link is less likely to be broken in 6 months.

Reply:
Updated.